src/main/webapp/js/pos.js

```diff
@@ -106,4 +106,12 @@ var sshBehave = {
     }
 };
 
-Behaviour.register(sshBehave);
\ No newline at end of file
+Behaviour.register(sshBehave);
+
+window.addEventListener("DOMContentLoaded", function() {
+    document.querySelectorAll(".pos-host-configs-data-holder").forEach(function(dataHolder) {
+        const { configName, effectiveDisableExec } = dataHolder.dataset;
+
+        exec_disabled[configName] = effectiveDisableExec === "true";
```

Comment:
There doesn't seem to be more correct way to convert string to boolean. We don't expect any values other than `"true"` or `"false"`, so this should be good enough.
Could alternatively use `JSON.parse(effectiveDisableExec)` here. Let me know if you like it more and I should change it.